core/market/src/protocol/discovery.rs

```diff
@@ -47,6 +47,7 @@ pub struct DiscoveryImpl {
 
     offer_queue: Mutex<Vec<SubscriptionId>>,
     unsub_queue: Mutex<Vec<SubscriptionId>>,
+    lazy_binder_queue: Mutex<Vec<String>>,
```

Comment:
This could be Option not Vec probably. Am I right?

Reply:
There is no limit on the times that `bind_gsb()` could be invoked, that's why I chose Vec.